pkg/component/component.go

```diff
@@ -937,6 +946,28 @@ func List(client *occlient.Client, applicationName string, localConfigInfo *conf
 	return compoList, nil
 }
 
+// List lists all components in active application
```

Comment:
`s/all/all s2i and devfile` would be more explanatory, IMO.

Reply:
done